packages/fiber/package.json

```diff
@@ -61,6 +61,7 @@
     "react": ">=18.0",
     "react-dom": ">=18.0",
     "react-native": ">=0.64",
+    "expo": ">=43.0",
```

Comment:
this should be optional I presume?

Reply:
Yes, although I'm not sure if we're supposed to have `expo-asset` and `expo-file-system` listed here as they're pulled in by `expo`. Still looking into how this should be installed under the hood as it might not be through NPM.